frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/EnhancedGraphTable.java

```diff
@@ -226,6 +231,53 @@ private void refreshModel(
     doOnUIThreadWhenReady.run();
   }
 
+  private Notification getSkippedBranchesNotification(IGitMacheteRepositorySnapshot repositorySnapshot,
+      GitRepository gitRepository) {
+    val notification = VcsNotifier.STANDARD_NOTIFICATION.createNotification(
+        format(getString("string.GitMachete.EnhancedGraphTable.skipped-branches-text"),
+            String.join(", ", repositorySnapshot.getSkippedBranchNames())),
+        NotificationType.WARNING);
+    notification.addAction(NotificationAction.createSimple(
+        () -> getString("action.GitMachete.EnhancedGraphTable.automatic-discover.slide-out-skipped"), () -> {
+          notification.expire();
+          slideOutBranch(repositorySnapshot, gitRepository);
+        }));
+    notification.addAction(NotificationAction.createSimple(
```

Comment:
Actually... not sure if this passes `GuiEffectChecker` successfully? I think the second param to `createSimple` should be marked as `@UI` (since the action is going to be executed on UI thread)

Reply:
![image](https://user-images.githubusercontent.com/19799111/109355385-6d029e80-787f-11eb-8d40-4707a5ba67e3.png)

Reply:
Haha okay ;)